scripts/ubuntu-jammy-from-community-gui/50-wayland_errrata.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+set -exv
+
+# init helpers
+helpers_dir=${MONOPACKER_HELPERS_DIR:-"/etc/monopacker/scripts"}
+for h in ${helpers_dir}/*.sh; do
+    . $h;
+done
+
+# see https://github.com/mozilla-platform-ops/monopacker/issues/138
+
+
+#
+# install tools
+#
+
+# used to modify specific blocks in .conf files
+apt install -y crudini
+
+
+#
+# dconf settings
+#
+
+# /etc/dconf/profile/user should have (no leading spaces):
+#
+#   user-db:user
+#   system-db:local
+cat > /etc/dconf/profile/user << EOF
+user-db:user
+system-db:local
+EOF
+
+# /etc/dconf/db/local.d/00-tc-gnome-settings should have (no leading spaces): 
+#
+# # /org/gnome/desktop/session/idle-delay
+# [org/gnome/desktop/session]
+# idle-delay=uint32 0
+#
+# # /org/gnome/desktop/lockdown/disable-lock-screen
+# [org/gnome/desktop/lockdown]
+# disable-lock-screen=true
+mkdir /etc/dconf/db/local.d/
+cat > /etc/dconf/db/local.d/00-tc-gnome-settings << EOF
+# /org/gnome/desktop/session/idle-delay
+[org/gnome/desktop/session]
+idle-delay=uint32 0
+
+# /org/gnome/desktop/lockdown/disable-lock-screen
+[org/gnome/desktop/lockdown]
+disable-lock-screen=true
+EOF
+
+# make dbus read the new configuration
+sudo dconf update
+
+# test
+ls -hal /etc/dconf/db/
+
+
+#
+# gdm3 settings  
+#
+
+# in [daemon] block of /etc/gdm3/custom.conf we need:
+#
+# XorgEnable=false
+
+crudini  --set /etc/gdm3/custom.conf daemon XorgEnable 'false'
+
+# verify/test
+cat /etc/gdm3/custom.conf
+echo "----"
+grep 'XorgEnable' /etc/gdm3/custom.conf
+grep 'XorgEnable' /etc/gdm3/custom.conf | grep false
+
+#
+# gdm3 service file
+#
+
+# copy /lib/systemd/system/gdm3.service to /etc/systemd/system and change its 
+# ExecStartPre to `/bin/sleep 15`
+
+# NOTES: this is a hack. alissy is working on a better solution.
+
+cp /lib/systemd/system/gdm3.service /etc/systemd/system/gdm3.service
```

Comment:
So since I went to the new solution https://github.com/mozilla-platform-ops/monopacker/pull/139/files#r1568497122, I have not been able to reproduce any new failure ? https://firefox-ci-tc.services.mozilla.com/tasks/groups/aKRzjXn2R5GNNrpt_8ZAIA

Reply:
Old hack removed and new gdm wait service added.